components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SQLActivityBrowser.java

```diff
@@ -225,15 +234,18 @@ public JsonObject getSocialObject(String targetId, String order,
 					tenantDomain, order, limit, offset);
 
 			activities = new ArrayList<Activity>();
-			while (resultSet.next()) {
-				JsonObject body = (JsonObject) parser.parse(resultSet
-						.getString(Constants.BODY_COLUMN));
-				int id = resultSet.getInt(Constants.ID_COLUMN);
-				Activity activity = new SQLActivity(body);
-				activity.setId(id);
-				activities.add(activity);
-			}
-			resultSet.close();
+            while (resultSet.next()) {
+                JsonObject body = (JsonObject) parser.parse(resultSet.getString(Constants.BODY_COLUMN));
+                int id = resultSet.getInt(Constants.ID_COLUMN);
+                Activity activity = new SQLActivity(body);
+                activity.setId(id);
+                activity.setLikeCount(getTotalLikes(id, 1));
+                activity.setDislikeCount(getTotalLikes(id, 0));
+                activity.setILike(isUserlikedActivity(tenantedUsername, id, 1));
+                activity.setIDislike(isUserlikedActivity(tenantedUsername, id, 0));
+                activities.add(activity);
+            }
+            resultSet.close();
```

Comment:
Thanks, I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/842/commits/b18db86eef4c71f1afc65b548dcaa4b5af055401)